Remove personal pronouns

Samyoul opened the pull request:
Resolves #115 

## Why make the change?

Technical documents should be written in formal unambiguous language, much of the specifications use a mixture of language modes.

## What has changed

- Removed use of personal pronouns
- Removed inappropriate use of [passive voice](https://en.wikipedia.org/wiki/English_passive_voice)
- Removed ambiguous subjects (mostly a consequence of removing passive voice)

Samyoul committed:
Rewrote 'you' and 'your' personal pronouns

Additionally I've made related sentences more concise. NOTE I've ignored usages of you and your in comments and in links and section titles

Samyoul committed:
Rewrote 'we' personal pronouns

Again not changing usages in comments

Samyoul committed:
Removed 'passive' and/or ambiguous language

Samyoul committed:
Added README information about language mode

Samyoul commented:
For details on the changes see [the addition to the README](https://github.com/status-im/specs/tree/update/personal-pronouns#language-mode)

Samyoul committed:
Added prepend to word list

decanus commented on README.md:
```diff
@@ -39,6 +39,67 @@ npm install
+### Language mode
+
+- Specifications SHOULD use formal technical language (*different from academic language*).
+- Were appropriate, language SHOULD NOT use personal pronouns.
```
```suggestion
- Where appropriate, language SHOULD NOT use personal pronouns.
```

Samyoul replied:
Ah, thank you. Good catch.

Samyoul committed:
Update README.md

Co-authored-by: Dean Eigenmann <7621705+decanus@users.noreply.github.com>

decanus commented on docs/draft/3-whisper-usage.md:
```diff
@@ -164,9 +162,9 @@ for i = 0; i < topicLen; i++ {
 Whisper is broadcast-based protocol. In theory, everyone could communicate using a single topic but that would be extremely inefficient. Opposite would be using a unique topic for each conversation, however, this brings privacy concerns because it would be much easier to detect whether and when two parties have an active conversation.
 
-Partitioned topics are used to broadcast private messages efficiently. By selecting a number of topic, it is possible to balance efficiency and privacy.
+Nodes use Partitioned topics to broadcast private messages efficiently. By selecting a number of topic, it is possible to balance efficiency and privacy.
```
```suggestion
Nodes use partitioned topics to broadcast private messages efficiently. By selecting a number of topic, it is possible to balance efficiency and privacy.
```

decanus commented on docs/draft/3-whisper-usage.md:
```diff
@@ -277,23 +275,23 @@ To exchange messages with client B, a client A SHOULD:
 - Send a message on client's B partitioned topic
 - Listen to the Negotiated Topic between A & B
-- Once a message is received from B, the Negotiated Topic SHOULD be used
+- Once client A receives a message from B, the Negotiated Topic SHOULD be used
```
```suggestion
- Once client `A` receives a message from `B`, the Negotiated Topic SHOULD be used
```

Samyoul committed:
Update docs/draft/3-whisper-usage.md

Co-authored-by: Dean Eigenmann <7621705+decanus@users.noreply.github.com>

Samyoul committed:
Update docs/draft/3-whisper-usage.md

Co-authored-by: Dean Eigenmann <7621705+decanus@users.noreply.github.com>

oskarth commented on README.md:
````diff
@@ -39,6 +39,67 @@ npm install
 npm run lint
 ```
 
+### Language mode
````
This part of the README is getting quite long... can we pull it out into its own STYLE/GUIDELINE md file? And include other related headers. Make README more scannable and friendly

oskarth commented on docs/draft/3-whisper-usage.md:
```diff
@@ -58,7 +56,7 @@ encryption properties to support asynchronous chat.
 * *Whisper network*: a group of Whisper nodes connected together through the internet connection and forming a graph
 * *Message*: decrypted Whisper message
```
missing

Samyoul replied:
Hey @oskarth, I'm sorry, I don't understand what is missing. Could you give me some more details? Thank you.

oskarth replied:
a decrypted

oskarth reviewed (approved):
Thanks! The guide looks to me. I didn't review all the changes in detail since there are quite a lot, but in general it looks good. I trust you followed your own guidelines :P

Samyoul committed:
Added missing 'a's

Samyoul committed:
Merge branch 'update/personal-pronouns' of github.com:status-im/specs into update/personal-pronouns

Samyoul committed:
Moved style related README info into STYLE-GUIDELINE.md

Samyoul committed:
Added reference to Google Technical Writing

Samyoul committed:
Tweaks to maintain consistency of changes across related specs

Samyoul committed:
Addressed spelling and added link to the Discord server

Samyoul commented:
Hey @decanus , I've done some tidying up, and I've addressed the changes you've suggested. Would you mind giving this PR another look over?

Note I've addressed something that Oskar asked me to do https://github.com/status-im/specs/pull/132#discussion_r433167487, which introduces a new `STYLE-GUIDELINE.md` for all things related to how writing should be done.

decanus commented:
@Samyoul I'll have another look tomorrow.

kdeme commented on docs/draft/12-sticker-pack.md:
```diff
@@ -41,25 +41,25 @@ The minimum sticker image resolution is 512x512, and its background SHOULD be tr
 
 ### Status sticker usage
-When a sticker is shown in the app, Status app makes an http GET request to IPFS gateway using the hyperlink. 
+When the app shows a sticker, Status app makes an http GET request to IPFS gateway using the hyperlink. 
```
```suggestion
When the app shows a sticker, the Status app makes an HTTP GET request to IPFS gateway using the hyperlink. 
```

kdeme commented on docs/draft/13-3rd-party.md:
```diff
@@ -61,7 +61,7 @@ Infura hosts a collection of full nodes for the Ethereum network and provides an
 
 ##### Concerns
-Making http request means that a user leaks metadata, which can be used in various attacks if the service is hacked.
+Making http request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```
```suggestion
Making a HTTP request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```

kdeme commented on docs/draft/13-3rd-party.md:
```diff
@@ -84,7 +84,7 @@ CryptoCompare is a service that shows live streaming prices, charts and analysis
 
 ##### Concerns
-Making http request means that a user leaks metadata, which can be used in various attacks if the service is hacked.
+Making http request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```
```suggestion
Making a HTTP request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```

kdeme commented on docs/draft/13-3rd-party.md:
```diff
@@ -97,7 +97,7 @@ There is a set of services that used for getting information about collectibles:
 
 ##### Concerns
-Making http request means that a user leaks metadata, which can be used in various attacks if the service is hacked.
+Making http request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```
```suggestion
Making a HTTP request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```

kdeme commented on docs/stable/11-waku-mailserver.md:
```diff
@@ -70,7 +70,7 @@ The envelope MUST be encrypted with a symmetric key agreed between the requester
 Historic messages MUST be sent to a peer as a packet with a P2P Message code (`0x7f`) followed by an array of Waku envelopes.
 
-In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allow to receive packets with the P2P Message code. By default, such packets are discarded.
+In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allowed to receive packets with the P2P Message code. By default, the node discards such packets.
```
I think the original text here was probably supposed to be something like: "that is **to** allow to receive packets with the P2P Message code"

So perhaps:
```suggestion
In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allowed to send packets with the P2P Message code. By default, the node discards such packets.
```

Samyoul replied:
Hmm, good point, really well spotted.

kdeme commented on docs/stable/4-whisper-mailserver.md:
```diff
@@ -67,7 +67,7 @@ The envelope MUST be encrypted with a symmetric key agreed between the requester
 Historic messages MUST be sent to a peer as a packet with a P2P Message code (`0x7f`) followed by an array of Whisper envelopes. It is incompatible with the original Whisper spec (EIP-627) because it allows only a single envelope, however, an array of envelopes is much more performant. In order to stay compatible with EIP-627, a peer receiving historic message MUST handle both cases.
 
-In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allow to receive packets with the P2P Message code. By default, such packets are discarded.
+In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allowed to receive packets with the P2P Message code. By default, the node discards such packets.
```
as in waku version

Samyoul committed:
Update docs/draft/12-sticker-pack.md

Co-authored-by: Kim De Mey <kim.demey@gmail.com>

Samyoul committed:
Update docs/draft/13-3rd-party.md

Co-authored-by: Kim De Mey <kim.demey@gmail.com>

Samyoul committed:
Update docs/draft/13-3rd-party.md

Co-authored-by: Kim De Mey <kim.demey@gmail.com>

Samyoul committed:
Update docs/draft/13-3rd-party.md

Co-authored-by: Kim De Mey <kim.demey@gmail.com>

Samyoul committed:
Update docs/stable/11-waku-mailserver.md

Co-authored-by: Kim De Mey <kim.demey@gmail.com>

Samyoul committed:
Addressed feedback

Samyoul committed:
Merge branch 'update/personal-pronouns' of github.com:status-im/specs into update/personal-pronouns
